src/view/buttons.py

```diff
@@ -0,0 +1,161 @@
+            Button.SHOW_MUSTWATCHES_BUTTON,
+            Button.DELETE_BUTTON
+        )
+
+    @classmethod
+    def markup_choose_user_add_item(cls,
+                                    *args: None) -> InlineKeyboardMarkup:
+        return cls.__markup_something(
+            2,
+            Button.ALL_BUTTON,
+            Button.ME_BUTTON,
+            Button.CHOOSE_USER_BUTTON
+        )
+
+    @classmethod
+    def markup_choose_specific_user(cls,
+                                    user_dict: Dict[int, str]) -> InlineKeyboardMarkup:
+        markup = InlineKeyboardMarkup()
+        markup.row_width = 1
+        buttons_list = list()
+        for key, value in user_dict.items():
+            buttons_list.append(InlineKeyboardButton(
+                text=value,
+                callback_data=Button.CHOOSE_SPECIFIC_USER_BUTTON_CALLBACK + str(key)
+            ))
+        buttons_list.append(InlineKeyboardButton(
+            text=Button.CHANGE_USER_REQUEST_BUTTON[0],
+            callback_data=Button.CHANGE_USER_REQUEST_BUTTON_CALLBACK
+        ))
+        markup.add(*buttons_list)
+        return markup
+
+    @classmethod
+    def markup_rate_or_delete_title(cls,
+                                    watches_dict: Dict[int, str]) -> InlineKeyboardMarkup:
+        markup = InlineKeyboardMarkup()
+        markup.row_width = 1
+        buttons_list = list()
+        for key, value in watches_dict.items():
+            buttons_list.append(InlineKeyboardButton(
+                text=value,
+                callback_data=Button.CHOOSE_TITLE_BUTTON_CALLBACK + str(key)
+            ))
+        buttons_list.append(InlineKeyboardButton(
+            text=Button.CHANGE_USER_REQUEST_BUTTON[0],
+            callback_data=Button.CHANGE_USER_REQUEST_BUTTON_CALLBACK
+        ))
+        markup.add(*buttons_list)
+        return markup
+
+    @classmethod
+    def markup_choose_or_add_title(cls,
+                                   watches_dict: Dict[int, str]) -> InlineKeyboardMarkup:
+        markup = InlineKeyboardMarkup()
+        markup.row_width = 1
+        buttons_list = list()
+        for key, value in watches_dict.items():
+            buttons_list.append(InlineKeyboardButton(
+                text=value,
+                callback_data=Button.CHOOSE_TITLE_BUTTON_CALLBACK + str(key)
+            ))
+        buttons_list.append(InlineKeyboardButton(
+            text=Button.ADD_NEW_MUSTWATCH_BUTTON[0],
+            callback_data=Button.ADD_NEW_MUSTWATCH_BUTTON_CALLBACK
+        ))
+        buttons_list.append(InlineKeyboardButton(
+            text=Button.CHANGE_USER_REQUEST_BUTTON[0],
+            callback_data=Button.CHANGE_USER_REQUEST_BUTTON_CALLBACK
+        ))
+        markup.add(*buttons_list)
+        return markup
+
+    @classmethod
+    def markup_rate_mustwatch(cls) -> InlineKeyboardMarkup:
+        markup = InlineKeyboardMarkup()
+        markup.row_width = 3
+        buttons_list = list()
+        for i in range(1, 10):
```

Comment:
There are lot of magic numbers. I have no idea what does they mean :sweat_smile:. Please create constants for them.